src/components/Panel/index.tsx

```diff
@@ -16,7 +15,7 @@ export interface PanelContentProps {
 export const Panel = styled.div`
   ${fullBleed};
   background-color: ${color.background};
```

Comment:
This might not be needed anymore without a shadow?

Reply:
Agreed, we do set the background but on the panel content. Checked, doesn't seem to do much now.